docs/UsersGuide/source/BuildingRunningTesting/RunSRW.rst

```diff
@@ -111,23 +111,37 @@ The first two steps depend on the platform being used and are described here for
 Load the Conda/Python Environment
 ------------------------------------
 
-The SRW App workflow is often referred to as the *regional workflow* because it runs experiments on a regional scale (unlike the *global workflow* used in other applications). The SRW App workflow requires installation of Python3 using conda; it also requires additional packages built in a separate conda evironment named ``workflow_tools``. On Level 1 systems, a ``workflow_tools`` environment already exists, and users merely need to load the environment. On Level 2-4 systems, users must create and then load the environment. The process for each is described in detail below.  
+The SRW App workflow requires a variety of Python packages. To manage the packages, the App relies
+on conda as a package manager and virtual environment manager. At build time users have the option
+to install the latest version of miniforge and automatically create the environments needed by SRW.
+Managed environments will no longer be updated on Tier I platforms for newer versions of SRW.
 
-.. _Load-WF-L1:
 
-Loading the Workflow Environment on Level 1 Systems
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Installing miniconda and creating environments
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In addition to building the SRW App components, the ``devbuild.sh`` script will install miniconda,
+specifically miniforge, and install the SRW-required environments if the ``conda`` target is
```

Comment:
Should probably remove mention of "conda" if "default" or nothing is now appropriate.

Reply:
Although I'm actually wondering if this section should be removed? The information is already in the BuildSRW chapter. We probably don't want people to rebuild by accident.

Reply:
I will remove the whole section. It is kind of in an odd spot now that this happens in devbuild.sh.